Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,132 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+
+namespace Fuse.Controls
+{
+	/**
+		TODO write docs please
+	*/
+	public class SystemScreenConfigBase : Behavior
+	{
+		public enum Visibility
+	    {
+	        None,
+	        Minimal,
+	        Full,
+	    }
```

Comment:
This is all white-space damanged. Please install an [EditorConfig](http://editorconfig.org/)-plugin for your text-editor, this will make white-space management much easier for you. This seems to be the case all over the place here, so please go through and clean up all of it.